libsolidity/formal/CHC.cpp

```diff
@@ -1807,6 +1807,17 @@ tuple<CheckResult, smtutil::Expression, CHCSolverInterface::CexGraph> CHC::query
 	CheckResult result;
 	smtutil::Expression invariant(true);
 	CHCSolverInterface::CexGraph cex;
+	if (m_settings.printQuery)
+	{
+		if (auto smtLibInterface = dynamic_cast<CHCSmtLib2Interface*>(m_interface.get()))
+		{
+			string smtLibCode = smtLibInterface->dumpQuery(_query);
+			m_errorReporter.info(
+				2339_error,
+				"; CHC: Requested query:\n" + smtLibCode
```

Comment:
```suggestion
				"CHC: Requested query:\n" + smtLibCode
```
Same here regarding `;`.